stable/hoppscotch/README.md

```diff
@@ -1,42 +1,16 @@
 # hoppscotch
 
-![Version: 0.2.7](https://img.shields.io/badge/Version-0.2.7-informational?style=flat-square) ![AppVersion: v1.9.7](https://img.shields.io/badge/AppVersion-v1.9.7-informational?style=flat-square)
+![Version: 0.2.8](https://img.shields.io/badge/Version-0.2.8-informational?style=flat-square) ![AppVersion: v1.9.9](https://img.shields.io/badge/AppVersion-v1.9.9-informational?style=flat-square)
 
 A free, fast and beautiful API request builder
 
 **Homepage:** <https://github.com/hoppscotch/hoppscotch>
 
-## How to install this chart
-
-Add Delivery Hero public chart repo:
-
-```console
-helm repo add deliveryhero https://charts.deliveryhero.io/
-```
-
-A simple install with default values:
-
-```console
-helm install deliveryhero/hoppscotch
-```
-
-To install the chart with the release name `my-release`:
-
-```console
-helm install my-release deliveryhero/hoppscotch
-```
-
-To install with some set values:
-
-```console
-helm install my-release deliveryhero/hoppscotch --set values_key1=value1 --set values_key2=value2
-```
-
-To install with custom values file:
```

Comment:
Can you run the docker command to generate the readme so it doesn't remove these instructions?

Details here: https://github.com/deliveryhero/helm-charts#contributing

Reply:
I did attempt to run this and commit the result. I'll try again.

Reply:
Pretty sure I ran it within the wrong directory the first time 🤦 .